src/components/course/data/tests/utils.test.jsx

```diff
@@ -884,9 +952,15 @@ describe('getMissingApplicableSubsidyReason', () => {
   it('returns SUBSCRIPTION_EXPIRED if there is an expired subscription', () => {
     const mockData = {
       ...baseMockData,
+      customerAgreement: {
+        availableSubscriptionCatalogs: ['test-catalog-uuid'],
```

Comment:
nit: For instances of `test-catalog-uuid` would we want to utilize the existing `mockCatalogUuid`